tests/rpc/rpc_client_test.go

```diff
@@ -378,9 +378,9 @@ func Test_DefaultClient_QueryGlobalStateByBlockHeight_StoredAddressableEntity_Pa
 	key := "package-8e08c43f144a13c915cf3681cc97bcd98c6a81d7b5da5164dc066318ec1c80a7"
 	res, err := client.QueryGlobalStateByBlockHeight(context.Background(), 1000, key, nil)
 	require.NoError(t, err)
-	assert.NotEmpty(t, res.StoredValue.Package)
-	assert.NotEmpty(t, res.StoredValue.Package.Versions)
-	assert.NotEmpty(t, res.StoredValue.Package.LockStatus)
+	assert.NotEmpty(t, res.StoredValue.SmartContract)
```

Comment:
@ZhmakAS , while the schema shows this renaming, the actual RPC response is still with `Package` property. Therefore, I would not include this change in this PR.